redundant-bmc/src/manager.cpp

```diff
@@ -70,13 +69,20 @@ sdbusplus::async::task<> Manager::startup()
         {
             co_await sibling->waitForSiblingUp(siblingTimeout);
 
-            if (previousRole == Role::Passive)
+            // Sibling service may have died.  Check again.
```

Comment:
I've looked at this for a while now, and I'm sure it's right, but it just feels like we're starting to work ourselves into the if/else wormhole. With tests now needed for every path and just a lot of complexity. Is that sibling service dying a real use case? Or just something that was only possible with some special error injections?

Reply:
It happened when the cfam daemon failed accessing the CFAM regs, which I think is a valid fail, and right now I just have the daemon crash when that happens.  I did it that way because I couldn't think of how else to alert rbmc manager that the sibling can't get any info from it.  If there would be another way for the rbmc manager to know that FSI is broken, then I wouldn't worry about this case here.  I'll think a bit more on it.